src/components/atoms/video/Video.tsx

```diff
@@ -1,21 +1,25 @@
-import React from 'react'
+import ReactPlayer from 'react-player'
 import { CardMedia } from '@mui/material'
 
 type VideoProps = {
   url: string
 }
 
-// WIP
 export function Video({ url }: VideoProps) {
   return (
-    <CardMedia
-      component={'iframe'}
-      src={addQueryParamsToVideoUrl(url)}
-      sx={{ aspectRatio: '16/9', border: '0' }}
-    />
+    <CardMedia component={'div'} sx={{ aspectRatio: '16/9', border: '0' }}>
+      <ReactPlayer
+        url={addQueryParamsToVideoUrl(url)}
+        controls
+        width="100%"
+        height="100%"
+      />
+    </CardMedia>
   )
 }
+
 function addQueryParamsToVideoUrl(url: string): string {
```

Comment:
🔧 I think we'll need to update this function to add query params based on the URL provided.  These query params are only for vimeo.